tests/Configuration/LoaderTest.php

```diff
@@ -5,46 +5,23 @@
 namespace Tests\SensioLabs\Deptrac\Configuration;
 
 use PHPUnit\Framework\TestCase;
-use SensioLabs\Deptrac\Configuration\Exception\FileCannotBeParsedAsYamlException;
-use SensioLabs\Deptrac\Configuration\Exception\FileDoesNotExistsException;
-use SensioLabs\Deptrac\Configuration\Exception\ParsedYamlIsNotAnArrayException;
+use SensioLabs\Deptrac\Configuration\Configuration;
 use SensioLabs\Deptrac\Configuration\Loader;
 
 /**
  * @covers \SensioLabs\Deptrac\Configuration\Loader
  */
 final class LoaderTest extends TestCase
 {
-    public function testLoadThrowsFileDoesNotExistsExceptionWhenFileDoesNotExist(): void
+    public function testLoadReturnsConfiguration(): void
     {
-        $file = __DIR__.'/../../examples/non-existent-file.yaml';
+        $file = __DIR__.'/../../examples/Uncovered.depfile.yaml';
```

Comment:
Please don't refer to an existing file in example dir. Sometimes I will change those examples files and I don't want break tests. It is fine when you are coping it to the fixtures dir.